packages/office-ui-fabric-react/src/components/Modal/Modal.scss

```diff
@@ -7,17 +7,20 @@
 // --------------------------------------------------
 // Modal styles
 
+:export { duration: $ms-duration2 }
```

Comment:
@mikewheaton  I found it yesterday via google, and it showed up in an issue on github. someone complained about it breaking webpack HMR, but I think we reload the page anyway for our build? /cc @dzearing